delfin/drivers/netapp/netapp_fas/netapp_fas.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from delfin.drivers import driver
+from delfin.drivers.netapp.netapp_fas import netapp_handler
+from delfin.drivers.netapp.netapp_fas.netapp_handler import NetAppHandler
+
+
+class NetAppFasDriver(driver.StorageDriver):
+
+    def __init__(self, **kwargs):
```

Comment:
missing entry point settings in setup.py

Reply:
Has been added